examples/index.html

```diff
@@ -51,10 +51,10 @@
         iframe.focus();
 
         var viewSrc = '<a href="https://github.com/iTowns/itowns/blob/master/examples/';
-        viewSrc += window.location.hash.substr(1);
+        viewSrc += window.location.hash.substr(1) || 'view_3d_map';
```

Comment:
The `code source` history is fixed?

Reply:
Unfortunately I couldn't make it work. Three.js, which has inspired this template, haven't find a solution too. It doesn't seems possible from what I've searched.